### PR TITLE
Renaming enterprise to on-premise (first pass)

### PR DESCRIPTION
**Pull request by Axxzilla**
Specifically
Enterprise Self-hosted = Self-Hosted Server
Enterprise Stand-Alone = Serverless

### CODE REVIEW

**Comment by jjsilva4** [comment]
Thanks Alex. Do you have a way of creating a sharable Local preview link of the website? Or was that just Diane who could do that?

**Comment by Axxzilla** [comment]
I don't have a way to easily do this - would have to research and try some things. When Diane did it, I believe she started Local and hosted locally, and added a clause to her hosts file to repoint traffic (after also updating and pulling the changes from the remote). 

It would be very nice if we had automation for this, but there is nothing in place. This is a lot of work and easier just to make the changes, push them live, then fix mistakes and push again. In no way is this ideal, but it is easier than trying to host using Local.

Miss our old automation for the support site :(

**Comment by jjsilva4** [comment]
My initial scan of the changes found nothing out of place, but yeah I'd have to check on production to confirm.

File: gitkraken-client/single-sign-on.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ taxonomy:
 
 ---
 
-GitKraken offers a Single Sign On (SSO) option as an easy way to sign in to your account. 
+GitKraken offers a Single Sign On (SSO) option as an easy way to sign in to your account. This is only for use with the cloud versions of GitKraken, it is not availibe for the serverless options.
```
Comment (amalyshkin) [inline gitkraken-client/single-sign-on.md:10]: "... is not availibe for ..." -> "... is not available for ..."

**Pull request merged**